### PR TITLE
feat: add `ARIMAPlus.coef_` property exposing `ML.ARIMA_COEFFICIENTS` functionality

### PR DESCRIPTION
**Pull request by SalemJorden**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: bigframes/ml/forecasting.py [hunk 1]
```diff
@@ -119,6 +119,35 @@ def predict(
             options={"horizon": horizon, "confidence_level": confidence_level}
         )
 
+    def arima_coefficients(
```
Comment (tswast) [inline bigframes/ml/forecasting.py:122]: We discussed exposing this as an `@property` called `coef_` to match sklearn.

File: bigframes/ml/core.py [hunk 2]
```diff
@@ -186,6 +186,11 @@ def arima_evaluate(self, show_all_candidate_models: bool = False):
 
         return self._session.read_gbq(sql)
 
+    def coef_(self) -> bpd.DataFrame:
```
Comment (tswast) [inline bigframes/ml/core.py:189]: Let's use the SQL name here too.

**Comment by tswast** [comment]
FYI: I updated the PR title to "feat: add `ARIMAPlus.coef_` property exposing `ML.ARIMA_COEFFICIENTS` functionality" to better reflect the user-facing changes.

Looks like the relevant tests are passing. :-) http://fusion2/ci/kokoro/prod%3Abigframes%2Fpresubmit%2Fe2e/activity/4cfd1e01-59bb-4955-8780-40c7fc914720

@GarrettWu @ashleyxuu The reason `coef_` was choses as a name is that this aligns with how regression models expose their coefficients in scikit-learn. See: https://stackoverflow.com/a/35386555/101923

**Review by tswast — APPROVED** [review]

File: bigframes/ml/sql.py [hunk 3]
```diff
@@ -318,6 +318,10 @@ def ml_evaluate(self, source_df: Optional[bpd.DataFrame] = None) -> str:
             return f"""SELECT * FROM ML.EVALUATE(MODEL `{self._model_name}`,
   ({source_sql}))"""
 
+    def ml_arima_coefficients(self) -> str:
```
Comment (ashleyxuu) [inline bigframes/ml/sql.py:321]: Could you help add the unit test for this method in https://github.com/googleapis/python-bigquery-dataframes/blob/main/tests/unit/ml/test_sql.py?

Comment (tswast) [reply bigframes/ml/sql.py:321]: +1 @SalemJorden please add such a test using the examples Ashley links here. Schedule a 1:1 with me to pair program if you need more support to get started.

**Review by tswast — CHANGES_REQUESTED** [review]

**Review by tswast — APPROVED** [review]

**Review by GarrettWu — APPROVED** [review]

**Pull request merged**